src/perf_tool_backend.ml

```diff
@@ -290,8 +290,19 @@ module Perf_line = struct
         in
         let time_hi = Int.of_string time_hi in
         let int64_of_hex_string str =
```

Comment:
move to a top-level function and add a couple unit tests?

Reply:
Sure, done